sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -183,6 +183,8 @@ object GpuCast extends Arm {
 
   val INVALID_NUMBER_MSG: String = "At least one value is either null or is an invalid number"
 
+  val INVALID_ROW_VALUE_MSG: String = "Error casting data on row"
```

Comment:
This is not used by `GpuCast`, it's only used by the tests and seems more appropriate there.